ansible/roles/proxy/tasks/main.yml

```diff
@@ -10,7 +10,40 @@
   register: apache_mirror
   failed_when: "'http' not in apache_mirror.stdout"
   changed_when: False
-- name: "download tarballs to proxy" 
+- name: "check if Accumulo tarball was uploaded to proxy"
```

Comment:
Were these separated out for the case where a snapshot tar ball already exists?

Could these three be collapsed into one loop?

Reply:
yes for the situation where the tarball exists because it was uploaded to the proxy.  Unfortunately, I don't think they can be collapsed as Ansible cannot do loops with multiple commands.